### PR TITLE
Add documentation about textclock format

### PR DESCRIPTION
**Pull request by xordspar0**

### CODE REVIEW

**Comment by Elv13** [comment]
Thanks for this!

File: lib/wibox/widget/textclock.lua [hunk 1]
```diff
@@ -28,9 +28,11 @@ if source_date_epoch and os.getenv("SOURCE_DIRECTORY") then
     end
 end
 
---- Set the clock's format
+--- Set the clock's format.  For information about the format specifiers, see
```
[the remark this answers is not in the document]
Comment (xordspar0) [reply lib/wibox/widget/textclock.lua]: Done. I don't think it changed the output html though.

**Comment by xordspar0** [comment]
Do you know if it's possible to render the URL as a link? It's just text currently.

**Comment by xordspar0** [comment]
Oh, it's just markdown. Got it.

**Comment by xordspar0** [comment]
Ok, now it's ready. :smile:

**Comment by Aire-One** [comment]
Oh I'm late here but can this be also ported to the constructor?

(And how can we fix the ldoc buggy render with the default value of the `format` argument on the constructor?)

**Comment by xordspar0** [comment]
> can this be also ported to the constructor?

I can make a PR for that.

> And how can we fix the ldoc buggy render with the default value of the format

I'll look into that as well.

**Comment by Aire-One** [comment]
> > And how can we fix the ldoc buggy render with the default value of the format
> 
> I'll look into that as well.

I already unsuccessfully checked that... Seems to be a limitation from the ldoc parsing algorithm. 

Hope you can find a solution.

**Comment by xordspar0** [comment]
I got it. See #2926.

**Pull request merged**